static/about.html

```diff
@@ -234,9 +235,20 @@ <h2 style="color: #416d9d;">Additional Comments</h2><br>
               similarly increase.
             </p>
             <p style="text-align: left">
-              There are other environmental and genetic factors not measured in a polygenic risk score that can
-              contribute to disease risk. Lifestyle factors, non-additive loci, epistasis, and gene by environment
-              interactions play a role in actual disease phenotype.
+	      Although polygenic risk scores have become increasingly prevalent in genetic research, historically, only
```

Comment:
This new information is fantastic! It is currently too text heavy however and should be broken up to match the style of the rest of the about page. Suggestions for breaking up the text while keeping the same words:
1. Put these two new paragraphs under their own heading above the additional comments section (name the new section "Standardization Practices" or something similar)
2. Put a link in for the Choi, et al. study and underline it
3. Make the Linkage disequilibrium paragraph a new paragraph (currently, it is squished with the paragraph above it as is shown in the attached screenshot)
4. Put the linkage disequilibrium paragraph under a sub heading (Linkage Disequilibrium)
5. Bullet point the ideas Choi has suggested
![additional comments too much text](https://user-images.githubusercontent.com/13890448/116331600-bdba4900-a795-11eb-932f-078614fbe562.PNG)